### PR TITLE
Draft a project idea for `GSoD`

### PR DESCRIPTION
**Pull request by mehabhalodiya**
Signed-off-by: mehabhalodiya mehabhalodiya@gmail.com

I am hereby adding the `gsod` directory inside mentorship for Google Season of Docs participation.
cc: @oleg-nenashev

### CODE REVIEW

File: mentorship/gsod/2022/project-ideas.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+in GSoD are avaialble on [this page](../README.md)
+
+## 2022 Project Ideas
+
+- [Documentation assessment](#keptn-docs-assessment) 
+
+<a name="keptn-docs-assessment"></a>
+### Documentation assessment
+
+This project is to have a docs assessment performed, to help us come with a plan for improving our documentaion. In particular, we're looking for someone to look at the content organization, the clarity of the language and concepts, and to make sure it's as readable as possible for both implementors and end users.
```
Comment (StackScribe) [inline mentorship/gsod/2022/project-ideas.md:15]:
```suggestion
This project would assess the Keptn documentation, to help us come with a plan for improving our documentation. In particular, we're looking for someone to look at the content organization, the clarity of the language and concepts, and to make sure it's as readable as possible for both implementors and end users.
```

**Comment by StackScribe** [comment]
Typo in PR title ;-)

File: mentorship/gsod/2022/project-ideas.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- More info: [tech docs](https://github.com/cncf/techdocs/blob/main/assessments/howto.md) and [criteria](https://github.com/cncf/techdocs/blob/main/assessments/criteria.md)
+- Slack channel for technical discussion: `#keptn-docs`
+- Areas to study/improve: Technical writing, Documentation, English, Git
+- Project size: 
+- Difficulty: Easy
+- Potential mentor(s):
```
Comment (StackScribe) [inline mentorship/gsod/2022/project-ideas.md:22]:
```suggestion
- Potential mentor(s): Meg McRoberts
```

**Review by StackScribe — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
This idea is fine with me! I am not 100% sure we would keep it in the final list, because there is a lot of evaluation being already done. But we can definitely start from it and expand the listing gradually

**Pull request merged**